user-guide/Cloud_Platform/Catalog/About_the_Catalog_module.md

```diff
@@ -25,6 +25,23 @@ There are several ways to access the Catalog module:
+| Scripted Connector | Contains a [DataMiner scripted connector](xref:Scripted_Connectors), enabling DataMiner to extract data from a third-party product for monitoring purposes. Deployment of the component will make it available on the target DataMiner System, where it can be used to create new elements. Scripted connectors provided by Skyline are subject to updates. If you want to benefit from these updates, you should therefore not alter these scripted connectors. |
+| Ad Hoc Data Source | Contains a [DataMiner ad hoc data source integration](xref:Configuring_an_ad_hoc_data_source_in_a_query), which allows you to make data from third-party products and various data sources available for ad hoc display in DataMiner low-code apps and dashboards. Data from an ad hoc data source is not continuously extracted from the third-party product, so it cannot be used for fault management, performance management, and various other background functions in DataMiner. Deploying the component will make it available on the target DataMiner System. Ad hoc data sources provided by Skyline are subject to updates. If you want to benefit from these updates, you should therefore not alter these ad hoc data sources. |
+| Data Transformer | Contains a data transformer, which allows you to transform data queried from DataMiner through a [GQI data query](xref:About_GQI) prior to serving it to users in low-code apps or dashboards. Deployment of the component will make it available on the target DataMiner System, where it can be used as is, as part of your GQI data queries, or where you can further adjust it to meet your specific needs. |
+| Data Query | Contains a [GQI data query](xref:About_GQI) to extract data from DataMiner, to transform it, and to serve it to users in low-code apps or dashboards. Deploying the component will make it available on the target DataMiner System, where it can be used as is, or where you can further adjust it to meet your own specific needs. |
+| Automation | Contains a general-purpose [DataMiner Automation script](xref:automation) to execute automation routines, triggered either on demand by users, based on a time schedule, or based on specific events that occur. Deploying the component will make it available on the target DataMiner System, where it can be used as is, or where you can further adjust it to meet your own specific needs. |
+| User-defined API | Contains a [user-defined API](xref:UD_APIs), which adds a new endpoint to your DataMiner System tailored to a specific use case, and which enables third-party applications to consume selective data from your DataMiner System. Deploying the component will make it available on the target DataMiner System, where it can be used as is, or where you can further adjust it to meet your own specific needs. You will then have to make that new user-defined API accessible by generating and sharing a token with the intended party. |
```

Comment:
I see that we use title casing for all the types except for this one. Wouldn't it be better to also use it here for the sake of consistency?
```suggestion
| User-Defined API | Contains a [user-defined API](xref:UD_APIs), which adds a new endpoint to your DataMiner System tailored to a specific use case, and which enables third-party applications to consume selective data from your DataMiner System. Deploying the component will make it available on the target DataMiner System, where it can be used as is, or where you can further adjust it to meet your own specific needs. You will then have to make that new user-defined API accessible by generating and sharing a token with the intended party. |
```